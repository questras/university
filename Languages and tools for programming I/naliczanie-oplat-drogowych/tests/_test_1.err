Error in line 2: A A 1
Error in line 5: Elka A1 74,5
