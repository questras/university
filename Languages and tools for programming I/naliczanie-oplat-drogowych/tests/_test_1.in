WA100 S3 123,7
A A 1
12P2000 S1 9,0
WA100 S3 125,2
Elka A1 74,5
WA100 A1 3,0
12P2000 S1 7,0
Elka S3 20,0
WA100 A1 14,1
Elka S3 10,0
WA100 A12 60,0
WA100 A12 61,0
WA100 A20 475,0
WA100 A20 475,0
ELKA S2 100,0
ELKA S2 99,0
SZ756 S7 13,3
?
WA100 A20 475,0
SZ756 S7 17,9
A20 A20 7,0
WA100 A20 470,0
A20 A20 8,0
?A20
?SZ756
