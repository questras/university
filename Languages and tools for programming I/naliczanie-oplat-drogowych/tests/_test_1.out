12P2000 S 2,0
ELKA S 1,0
Elka S 10,0
WA100 A 12,1 S 1,5
A1 11,1
S1 2,0
S2 1,0
S3 11,5
A12 1,0
A20 0,0
A20 A 1,0
A20 6,0
SZ756 S 4,6
